test_scripts/API/VehicleData/StabilityControlsStatus/019_UnsubscribeVehicleData_StabilityControlsStatus_disallowed_by_policy.lua

```diff
@@ -0,0 +1,36 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0253-New-vehicle-data-StabilityControlsStatus.md
+--
+-- Description: UnsubscribeVehicleData RPC with `stabilityControlsStatus` parameter only is NOT allowed by Policies
+--
+-- Precondition:
+-- 1) SDL and HMI are started
+-- 2) App is registered
+-- 3) PTU is successfully performed
```

Comment:
@LitvinenkoIra Add comment please what is performed in PTU (minimal) here or just before `common.ptUpdateMin`

Reply:
@AByzhynar Done in 4689ec2